### PR TITLE
Added the vm creation script and unit tests

### PR DESCRIPTION
**Pull request by RJaliliSTFC**
Script to create vms on every hv in every aggregate

### CODE REVIEW

File: Openstack-VM-Creation-Script/vm_creation_v2.py [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+    for agg_name, flav in aggregate_flavor_list.items():
+        hvs = {agg_name: hv_list[agg_name]}
+        dict_of_aggregate_to_hv_objects.update(hvs)
+    return dict_of_aggregate_to_hv_objects
+
+
+def create_vms_on_hvs_with_space(hv_list, aggregate_flavor_list, image):
+    created_list = []
+    security_group_id = "771352ca-41b7-4fb6-addc-882aa0d1d628"
+    network_id = "fa2f5ebe-d0e0-4465-9637-e9461de443f1"
```
Comment (meoflynn) [inline Openstack-VM-Creation-Script/vm_creation_v2.py:112]: Are these meant to be hard coded into the method? 
If they are needed could you add a comment to each one about what security group and network goes with the ids?

File: Openstack-VM-Creation-Script/vm_creation_v2.py [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+
+    current_image = select_single_image(image)
+
+    print(
+        create_vms_on_hvs_with_space(
+            dict_of_flavor_to_hypervisor_objects,
+            aggregate_to_smallest_flavor_dict,
+            current_image,
+        )
+    )
```
Comment (meoflynn) [inline Openstack-VM-Creation-Script/vm_creation_v2.py:164]: Could you add a comment about what this does?

**Review by meoflynn — CHANGES_REQUESTED** [review]
Looks good so far, could you add docstrings to methods in the script and the unittests to describe what each method does?

File: Openstack-VM-Creation-Script/vm_creation_v2.py (hunk 1 above)
Comment (RJaliliSTFC) [reply Openstack-VM-Creation-Script/vm_creation_v2.py:112]: Added a comment for their ID's, not been discussed if they should be a variable or not, they are also currently dev-cloud IDs which I will change as I don't think the script was meant to be run for the dev-cloud

**Comment by DavidFair** [comment]
We should copy over / adapt one of the workflows to run pylint, black and the unit tests that we now have

**Comment by DavidFair** [comment]
Can we close this since we've moved it to the st2 pack?

**Pull request closed without merging**